app/src/test/ActionCreatorTestHelpers.ts

```diff
@@ -1,7 +1,8 @@
 import {createMockStore} from "./mocks/mockStore";
```

Comment:
From this point onwards in the PR all changes are just migrating from chai/mocha syntax to jest.